### PR TITLE
multiple updates from basecamp

### PR DESCRIPTION
**Pull request by dmgenitrust**

### CODE REVIEW

**Comment by rgenito** [comment]
@dmgenitrust heya Danny! there are many conflicts... will need you to make sure you pull down the branch(es) and merge before creating the PR request. Thank you!

File: htdocs/buying.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
             </div>
             <ul class="nav navbar-nav">
                 <li class="active"><a href="#">Buy Crypto</a></li>
-                <li><a href="/sell.html">Sell Crypto</a></li>
+                <li><a href="/selling.html">Sell Crypto</a></li>
```
Comment (rgenito) [inline htdocs/buying.html]: Keep this as "sell.html" -- we've already announced this link to the DASH team and they're announcing it to the community... better to revert this.

File: htdocs/index.html [hunk 2]
```diff
@@ -32,9 +32,9 @@ <h1 class="text-center">Wall of Coins Reference Client</h1>
     <div class="container">
         <div class="row">
             <div class="col-md-2"></div>
-            <div class="col-md-3"><a class="btn btn-lg btn-block btn-primary" href="/buy.html">Buy</a></div>
+            <div class="col-md-3"><a class="btn btn-lg btn-block btn-primary" href="/buying.html">Buy</a></div>
```
Comment (rgenito) [inline htdocs/index.html]: Keep the original links for this too.

File: htdocs/main.js [hunk 3]
```diff
@@ -181,6 +184,8 @@
         } else {
```
Comment (rgenito) [inline htdocs/main.js]: Recommended format:
}
else {

File: htdocs/main.js [hunk 4]
```diff
@@ -125,7 +128,7 @@
     function orderDynamicFields(payFields){
         var ordered = _.orderBy(payFields.dynamicFields, 'displaySort', 'asc');
         var confirms = [];
-        _.each(payFields.dynamicFields, function(payField){
+        _.each(ordered, function(payField){
             confirms.push(payField);
             var confirm = _.find(payFields.confirmFields, function(o) { return o.name == payField.name; });
             if (confirm){
```
Comment (rgenito) [inline htdocs/main.js]: recommended format: "if (confirm) {" (notice extra space)

File: htdocs/main.js [hunk 5]
```diff
@@ -471,11 +476,20 @@
                 }
             });
         },
+        geolocation: function () {
+            if (navigator.geolocation) {
+                navigator.geolocation.getCurrentPosition(savePosition);
+            }else{
```
Comment (rgenito) [inline htdocs/main.js]: }
else {

File: htdocs/selling.html [hunk 6]
```diff
@@ -12,7 +13,7 @@
                 <a class="navbar-brand" href="/">WOC API <sup><small>DEMO</small></sup></a>
             </div>
             <ul class="nav navbar-nav">
-                <li><a href="/buy.html">Buy Crypto</a></li>
+                <li><a href="/buying.html">Buy Crypto</a></li>
```
Comment (rgenito) [inline htdocs/selling.html]: last item to keep as "buy.html"? :)

**Review by robertgenito — CHANGES_REQUESTED** [review]
"buy.html" and "sell.html" resources names must stay the same. Everything else is lookin' good!

**Pull request merged**